packages/shared/providers.yaml

```diff
@@ -753,7 +755,7 @@ gong-oauth:
     disable_pkce: true
     token_request_auth_method: basic
     proxy:
-        base_url: https://api.gong.io
+        base_url: ${connectionConfig.api_base_url_for_customer}
```

Comment:
Is there a way to fallback to `https://api.gong.io` if the connection doesn't have the right connection config? 

Would something like `${connectionConfig.api_base_url_for_customer || 'https://api.gong.io}` work?

Reply:
@bastienbeurier , I will do some test on this and see if the platform will handle such case. Incase of any changes to the platform, I will add them to this pull request. Thank you.

Reply:
For reference, I checked that no one was using Gong in prod at the moment.

Reply:
It looks like people are using this in prod now. Is there a way to have a fallback system? 

Something like `${connectionConfig.api_base_url_for_customer || 'https://api.gong.io'}`